src/components/CourseOverviewListItem/CourseOverviewListItem.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import ClockIcon from "../../assets/images/clock-icon.svg";
+import "./CourseOverviewListItem.scss";
+import Redo from "../../assets/images/redo.svg";
+import { Link } from "react-router-dom";
+
+const CourseOverviewListItem = ({ image, title, duration, id, category }) => {
+  const [completed, setCompleted] = useState(false);
+
+  const handleClick = () => {
+    setCompleted(true);
+  };
+
+  const setImage = () => {
+    if (completed === true) {
+      return Redo;
+    } else {
+      return image;
+    }
+  };
+
+  const setPath = () => {
+    if (category === "lesson") {
+      return "/lesson/:lessonId";
```

Comment:
This will need to use the `id` prop. At the moment it goes to  "/lesson/:lessonId" when you click.